code-challenges/sort/challenges-03.test.js

```diff
@@ -0,0 +1,285 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named sortBackwards that takes in an array of numbers and returns the same array, with the numbers sorted, highest to smallest.
+------------------------------------------------------------------------------------------------ */
+
+const sortBackwards = (arr) => {
+  // Solution code here...
+  arr.sort( (a,b) => (b-a));
+  return arr;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named alphabetize that takes in an array of strings and returns the same array with the strings sorted alphabetically.
+
+In this alphabetization, capital letters come before lower case letters.
+
+For example, ['Alphabet', 'Zebra', 'alphabet', 'carrot'] is correctly sorted.
+------------------------------------------------------------------------------------------------ */
+
+const alphabetize = (arr) => {
+  // Solution code here...
+  arr.sort();
```

Comment:
This looks great! If you had wanted you could have combined lines 27 and 28 into one:

return arr.sort();